### PR TITLE
PP-520 sketch & sketch large updates

### PR DESCRIPTION
**Pull request by alanbjorklund**
PP-520

minor updates to sketch & sketch large
mark these printers public
ran 20 prints in the NYC office to validate the print quality

### CODE REVIEW

File: resources/definitions/ultimaker_sketch.def.json [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+        "machine_name": { "default_value": "MakerBot Sketch" },
+        "machine_nozzle_cool_down_speed": { "default_value": 0.8 },
+        "machine_nozzle_heat_up_speed": { "default_value": 1.4 },
+        "machine_start_gcode": { "default_value": "M140 S50 T0; Set Platform Temp\nM104 S220 T0; Set Extruder Temp\nG90; Use Absolute Positioning\nG28; Home\nM132 X Y Z A B; Set Current Position to Home\nG161 X Y F3300; Move to min axes positions\nM7 T0; Wait For Platform to Heat\nM6 T0; Wait For Extruders to Heat\nM651; Turn on back fan\nM907 X100 Y100 Z40 A80 B20; Set Stepper Currents\nM106; Enable Cooling Fan\n; Purge Line\nG92 E0; Reset Extruder Axis Position\nG1 X-26.18 Y-75.90 Z0.200 F420\nG1 X26.18 Y-75.90 E10\nG92 E0; Reset Extruder Axis Position\n; Start GCode\n" },
+        "machine_width": { "default_value": 150 },
+        "material_diameter": { "default_value": 1.75 },
+        "material_flow": { "default_value": 100 },
+        "min_bead_width":
+        {
+            "default_value": 0.4,
```
Comment (wawanbreton) [inline resources/definitions/ultimaker_sketch.def.json:125]: This default value will be ignored as you also provide a value (fails unit test)
```suggestion
```

File: resources/definitions/ultimaker_sketch.def.json [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+            "minimum_value": "line_width * 0.5",
+            "minimum_value_warning": "line_width * 0.75",
+            "value": "line_width"
+        },
+        "min_wall_line_width":
+        {
+            "default_value": 0.4,
```
Comment (wawanbreton) [inline resources/definitions/ultimaker_sketch.def.json:132]: Same here
```suggestion
```

**Review by wawanbreton — CHANGES_REQUESTED** [review]
Mesh files are missing, and some setting definitions issues pointed out by unit tests

File: resources/definitions/ultimaker_sketch.def.json [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+    "version": 2,
+    "name": "MakerBot Sketch",
+    "inherits": "ultimaker",
+    "metadata":
+    {
+        "visible": true,
+        "author": "Ultimaker",
+        "manufacturer": "Ultimaker B.V.",
+        "file_formats": "application/x-makerbot-sketch",
+        "platform": "ultimaker_sketch_platform.obj",
```
[the remark this answers is not in the document]
Comment (alanbjorklund) [reply resources/definitions/ultimaker_sketch.def.json:11]: fixed in latest commit

File: resources/definitions/ultimaker_sketch_large.def.json [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    "version": 2,
+    "name": "MakerBot Sketch Large",
+    "inherits": "ultimaker_sketch",
+    "metadata":
+    {
+        "visible": true,
+        "author": "Ultimaker",
+        "manufacturer": "Ultimaker B.V.",
+        "file_formats": "application/x-makerbot-sketch",
+        "platform": "ultimaker_sketch_large_platform.obj",
```
[the remark this answers is not in the document]
Comment (alanbjorklund) [reply resources/definitions/ultimaker_sketch_large.def.json:11]: fixed in latest commit

File: resources/definitions/ultimaker_sketch.def.json (hunk 1 above)
Comment (alanbjorklund) [reply resources/definitions/ultimaker_sketch.def.json]: fixed in latest commit

File: resources/definitions/ultimaker_sketch.def.json (hunk 2 above)
Comment (alanbjorklund) [reply resources/definitions/ultimaker_sketch.def.json]: fixed in latest commit

**Pull request merged**